src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests.cs

```diff
@@ -12,6 +12,7 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests
 {
     public partial class IOperationTests : SemanticModelTestBase
     {
+        [CompilerTrait(CompilerFeature.IOperation)]
```

Comment:
I haven't seen trail on class. I think majority of roslyn tests' traits are on each test methods.